admin_app/src/components/ContentModals.tsx

```diff
@@ -0,0 +1,144 @@
+// ConfirmDelete.jsx
+import React, { useState, useEffect } from "react";
+import { Content } from "./ContentCard";
+import { XMarkIcon } from "@heroicons/react/20/solid";
+import TextareaAutosize from "react-textarea-autosize";
+
+interface ConfirmDeleteProps {
```

Comment:
Good idea to specify type or interface to take full advantage of typing in typescript